ReadingList.Notes.Logic/Models/BookRecord.cs

```diff
@@ -8,12 +8,14 @@ public class BookRecord
         public string Title { get; }
         public IEnumerable<string> Authors { get; }
         public IEnumerable<string> Notes { get; }
+        public string RawFileName { get; }
 
-        public BookRecord(string title, IEnumerable<string> authors, IEnumerable<string> notes)
+        public BookRecord(string title, IEnumerable<string> authors, IEnumerable<string> notes, string rawFileName)
         {
             Title = title;
             Authors = authors;
             Notes = notes;
+            RawFileName = rawFileName;
```

Comment:
Should consider splitting up to it's own models here so we can have the SHA-hash and raw filename without bleeding out to the external API.